js-exercises/telephone-check/telephoneCheck.js

```diff
@@ -0,0 +1,9 @@
+function telephoneCheck(...args) {
+  var PhoneNumber = args.toString()
+  
+  if (PhoneNumber.match(/[1-9]/g).length==11){return true}
```

Comment:
Here you could have used variable for regex. Example: const phoneNumberRegex = /[1-9]/g